docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,242 @@ grand_parent: "General"
 ---
 
 # Getting Started With bpmn-js
+
+bpmn-js is a [BPMN 2.0](https://www.omg.org/spec/BPMN/2.0/About-BPMN/) rendering toolkit and web modeler. It allows to view and edit BPMN 2.0 diagrams in the browser without any backend requirement.
```

Comment:
What about adding also the second part of the sentence from the walkthrough 
`It is written in JavaScript, embeds BPMN 2.0 diagrams into modern browsers and requires no server backend. That makes it easy to embed it into any web application.`

Reply:
Done 👍